mne/tests/test_cov.py

```diff
@@ -82,8 +82,8 @@ def test_cov_order():
     ch_names = [info['ch_names'][pick]
                 for pick in pick_types(info, meg=False, eeg=True)]
     cov = read_cov(cov_fname)
-    with warnings.catch_warnings(record=True):  # no avg ref present
```

Comment:
the purpose is to get rid of spurious warnings in the tests. Warnings that are there just because we want to gain testing time.